services/storage/src/simcore_service_storage/exceptions/errors.py

```diff
@@ -45,3 +44,7 @@ class InvalidFileIdentifierError(AccessLayerError):
 
 class DatCoreCredentialsMissingError(StorageRuntimeError):
     msg_template: str = "DatCore credentials are incomplete. TIP: Check your settings"
+
+
+class SelectionNotAllowedError(StorageRuntimeError):
+    msg_template: str = "Selection='{selection}' must share the same parent folder"
```

Comment:
does that mean they must be in the same project? if that is the case, that would be clearer for the user.

Reply:
No, it can be different projects but you can't pick a file in a project and a file in a different project. It needs to be from the same level in the same directory. I'll rephrase it like that